### ISSUE TITLE
How can I use pretrained weights uploaded on GoogleDrive? 

### ISSUE THREAD

**Issue by KaSangeun** [Dootmaan/DSRL#24]
Hi ! 
Thank you for sharing your DSRL codes. 

I wonder how I can use retrained weights uploaded on your GoogleDrive. 
Do I just download it and type train.py --resume 'path/to/weights' into the terminal ?

**Comment by Dootmaan** [comment]
Hi @KaSangeun and thank you for your question. After you download the weights, please find the path to you weights (let's assume the path is "/home/kasangeun/your_downloaded_weights.pt") and run our code with
```
python3 -u train.py --resume /home/kasangeun/your_downloaded_weights.pt
```

**Comment by KaSangeun** [comment]
Thank you for the speedy reply and it finally works now ! Thank you for your help :)

I have one more question, when I trained the code, I can see the Loss value came out over 500 per epoch. (As shown in the attached pic)
Is that normal Loss value from DSRL? 
<img width="1411" alt="스크린샷 2024-05-21 오후 1 06 36" src="https://github.com/Dootmaan/DSRL/assets/109417984/0d9f8d66-22c1-4025-901a-0d70f4042018">

**Comment by Dootmaan** [comment]
> Thank you for the speedy reply and it finally works now ! Thank you for your help :)
> 
> I have one more question, when I trained the code, I can see the Loss value came out over 500 per epoch. (As shown in the attached pic) Is that normal Loss value from DSRL? <img alt="스크린샷 2024-05-21 오후 1 06 36" width="1411" src="https://private-user-images.githubusercontent.com/109417984/332279568-0d9f8d66-22c1-4025-901a-0d70f4042018.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYyNjc0MTUsIm5iZiI6MTcxNjI2NzExNSwicGF0aCI6Ii8xMDk0MTc5ODQvMzMyMjc5NTY4LTBkOWY4ZDY2LTIyYzEtNDAyNS05MDFhLTBkNzBmNDA0MjAxOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyMVQwNDUxNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mNTQ5OWYyNGYxOTI1YjI1ODFkNjgyNTU4NDNjMjgyNGNhMjM2N2RiZTMzOGUyNzdiZTgyMDJmMmMxNmFlM2ZlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.pzUMEePZTlMd80Z60INBjqKvXnXb7OfePDIs_1n-VS0">

That seems unusual to me, considering that the validation performance is still good. Maybe you could print the several losses separately to check which one has probably gone wrong. You could also try to train from scratch to see whether the loss will change.

**Comment by KaSangeun** [comment]
Thanks for the helpful advice. I'll try it !

**Comment by KaSangeun** [comment]
Hello !
I have a question during training DSRL code.
I am curious about the training conditions of the 2020-12-15 pretrained model including the dataset, input size, which backbone network you used, etc.
Could you please let me know ?
